views/pages/practitioner/declaration.njk

```diff
@@ -0,0 +1,354 @@
+                </tr>
+                <tr class="odd:bg-gray-300">
+                    <th scope="row" class="w-1/3 text-left font-normal p-4 border-2 border-white align-top">
+                        {{ __('treatments.form.results.desc') }}
+                    </th>
+                    <td class="w-2/3 p-4 border-2 border-white align-top">{{ treatment.treatmentResults }}</td>
+                </tr>
+                <tr class="odd:bg-gray-300">
+                    <th scope="row" class="w-1/3 text-left font-normal p-4 border-2 border-white align-top">
+                        {{ __('treatments.form.treated_conditions.question') }}
+                    </th>
+                    <td class="w-2/3 p-4 border-2 border-white align-top">
+                        {% for condition in treatment.Conditions %}
+                            {{ condition.conditionName }}<br>
+                        {% endfor %}
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        {% endfor %}
+    </details>
+
+    <details class="mt-4 border-none" open>
+        <summary>
+            <h2 class="mt-4 text-4xl inline">{{ __('dashboard.future-work-capacity') }}</h2>
+            <div class="float-right">
+                <a href="{{ route('work') }}">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" class="inline-block h-4 w-4"><path d="M17.561 2.439c-1.442-1.443-2.525-1.227-2.525-1.227L8.984 7.264 2.21 14.037 1.2 18.799l4.763-1.01 6.774-6.771 6.052-6.052c-.001 0 .216-1.083-1.228-2.527zM5.68 17.217l-1.624.35a3.71 3.71 0 0 0-.69-.932 3.742 3.742 0 0 0-.932-.691l.35-1.623.47-.469s.883.018 1.881 1.016c.997.996 1.016 1.881 1.016 1.881l-.471.468z"/></svg>
+                    {{ __('edit') }}
+                </a>
+            </div>
+        </summary>
+
+        <table class="table-fixed">
+            <tbody class="bg-gray-100">
+                <tr class="odd:bg-gray-300">
+                    <th scope="row" class="w-1/3 text-left font-normal p-4 border-2 border-white align-top">{{ __('section6.return_work_question', data.fullName) }}</th>
+                    <td class="w-2/3 p-4 border-2 border-white align-top">
+                        {{ data.returnToWorkText }}
+                        {% if data.returnToWork === 1 %}
+                            - {{ data.returnToWorkWhenText }}
+                        {% endif %}
+                    </td>
+                </tr>
+                {% if data.returnToWork === 1 %}
+                    <tr class="odd:bg-gray-300">
+                        <th scope="row" class="text-left font-normal p-4 border-2 border-white align-top">{{ __('section6.type_work_question', data.fullName) }}</th>
+                        <td class="p-4 border-2 border-white align-top">{{ data.typeOfWorkText }}</td>
+                    </tr>
+                {% endif %}
```

Comment:
We should include the workDetails field here too, probably

Reply:
done